foursight_core/react/api/react_routes.py

```diff
@@ -14,7 +15,7 @@ def __init__(self):
     # ----------------------------------------------------------------------------------------------
 
     @route("/{env}/auth0_config", authorize=False)
-    def reactapi_route_auth0_config(env) -> Response:
+    def reactapi_route_auth0_config(env) -> Response:  # noqa
```

Comment:
Oh it's because it's not marked as @staticmethod and has no self argument. With the Chalice route decorator (which I've wrapped) it is implicitly static.

Reply:
Added comments to the noqa.